packages/client/src/views/GrantDetails.vue

```diff
@@ -461,12 +460,16 @@ export default {
   column-gap: 90px;
   row-gap: 48px;
   .grant-details-back-link {
-    padding-top: 20px;
-    font-size: 1rem;
+    margin-top: 24px;
+    font-size: 14px;
+    font-weight: 700;
+    color: #6D7278;
     a {
+      color: #6D7278;
       cursor: pointer;
-      color: black;
-      text-decoration: none;
+      &::hover {
+        text-decoration: underline !important;
```

Comment:
We should try to avoid using `!important` when possible, as it tends to make maintaining things more difficult over time. In general, if you need to override a bootstrap-provided css directive, you can simply make your own CSS directive more specific so it takes precedence.